cmd/kubeadm/app/constants/constants_windows.go

```diff
@@ -23,5 +23,5 @@ const (
 	DefaultDockerCRISocket = "npipe:////./pipe/docker_engine"
 
 	// PauseVersion indicates the default pause image version for kubeadm
-	PauseVersion = "1.4.0"
+	PauseVersion = "3.4.1"
```

Comment:
i'm assuming we have to change this too?
https://github.com/kubernetes/kubernetes/blob/d966e0b7fcbe5d0b386f1d99d369841903164599/cmd/kubeadm/app/images/images_windows.go#L33

Reply:
if this is a manifest list image now (multi-arch) we should do the following in kubeadm:

remove these files:
https://github.com/kubernetes/kubernetes/blob/d966e0b7fcbe5d0b386f1d99d369841903164599/cmd/kubeadm/app/images/images_unix.go
https://github.com/kubernetes/kubernetes/blob/d966e0b7fcbe5d0b386f1d99d369841903164599/cmd/kubeadm/app/images/images_windows.go

add this function:
```
// GetPauseImage returns the image for the "pause" container
func GetPauseImage(cfg *kubeadmapi.ClusterConfiguration) string {
	return GetGenericImage(cfg.ImageRepository, "pause", constants.PauseVersion)
}
```

back to:
https://github.com/kubernetes/kubernetes/blob/d966e0b7fcbe5d0b386f1d99d369841903164599/cmd/kubeadm/app/images/images.go

Reply:
will test it and refactor this if needed